code/modules/antagonists/gang/outfits.dm

```diff
@@ -17,14 +17,15 @@
 	suit = null
 	shoes = /obj/item/clothing/shoes/combat/swat
 	gloves = null
-	glasses = /obj/item/clothing/glasses/hud/spacecop
+	glasses = /obj/item/clothing/glasses/sunglasses
 	ears = /obj/item/radio/headset/headset_sec/alt
 	mask = null
 	head = /obj/item/clothing/head/spacepolice
 	belt = /obj/item/gun/ballistic/automatic/pistol/m1911
 	r_pocket = /obj/item/lighter
 	l_pocket = /obj/item/restraints/handcuffs
 	id = /obj/item/card/id
+	r_hand = /obj/item/gun/energy/disabler
```

Comment:
would it be possible to give them telebatons or something as well, just so that they'll still have a nonlethal option (that doesn't rely on shovestuns) once they're out of disabler charge?

Reply:
no

Reply:
as in, i won't be doing this